apps/articles/client/team_channel.coffee

```diff
@@ -0,0 +1,43 @@
+Backbone = require 'backbone'
+_ = require 'underscore'
+Articles = require '../../../collections/articles.coffee'
+Channel = require '../../../models/channel.coffee'
+ArticlesGridView = require '../../../components/articles_grid/view.coffee'
+initCarousel = require '../../../components/merry_go_round/horizontal_nav_mgr.coffee'
+sd = require('sharify').data
+
+module.exports.TeamChannelView = class TeamChannelView extends Backbone.View
+
+  initialize: ->
+    @channel = new Channel sd.CHANNEL
+    @gridArticles = new Articles
+    @gridArticles.url = "#{@gridArticles.url}/?&published=true&limit=12&sort=-published_at&channel_id=#{@channel.get('id')}"
```

Comment:
You can pass these query params into a fetch call like `gridArticles.fetch(data: { published: true, limit: 12 ... })`. That, or pulling in the querystring module could help clean this up. Not a blocker by any means though.